modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -78,6 +78,21 @@ const Modal = ({
 	});
 
 	const onButtonClick = ({formId, onClick, type}) => {
+		const submitForm = (form) => {
+			if (form.requestSubmit) {
+				form.requestSubmit();
+			}
+			else {
+				const canceled = form.dispatchEvent(
+					new Event('submit', {cancelable: true})
+				);
+
+				if (canceled) {
```

Comment:
@victorg1991 Should we consider changing this variable name? From what I understand, the value will be `true` if the event is _not_ cancelled, and `false` if it is cancelled. Maybe name it something like `completed`? I'm open to suggestions.

Reply:
Nice catch! `accepted`?

Reply:
Sounds great!

Reply:
Nice, I'll change that :)